File: Code Chef/C Programming/800-899/840 Is it a VOWEL or CONSONANT VOWELTB.c
#include <stdio.h>
int main(void)
{
	char t;
	scanf("%c",&t);
	if(t=='A'||t=='E'||t=='I'||t=='O'||t=='U')
	{
	    printf("Vowel\n");
	}
	else
	{
	    printf("Consonant\n");
	}
	return 0;
}

File: Code Chef/JavaScript/800-899/840 Is it a VOWEL or CONSONANT VOWELTB.js
let input = '';
process.stdin.on('data',(data) => input += data);

process.stdin.on('end', () => {
    const char = input.trim();
    const vowels = ['A','E', 'I', 'O', 'U'];
    
    (vowels.includes(char)) ? console.log('Vowel') : console.log('Consonant');
});
